vivisect/qt/funcgraph.py

```diff
@@ -111,8 +111,14 @@ def _renderMemoryCallback(self, cb, data):
         runner = functools.partial(self._renderMemoryFinish, cb)
         e_memcanvas.MemoryCanvas.renderMemory(self, va, size, cb=runner)
 
-    def renderMemory(self, va, size, cb):
+    def renderMemory(self, va, size, cb=None):
+        '''
+        Funcgraph specific renderMemory() function.
+        '''
         # For the funcgraph canvas, this will be called once per code block
+        if not cb:
```

Comment:
When does this happen?

Reply:
i should have known you'd ask.  i don't recall.  i recently had it happen.  sorry, it's not related to viv_loader.  i'll try to track it down.